### DISCUSSION TITLE
#31 togETHer meeting - Jan 24 2022

### DISCUSSION THREAD

**Discussion by bobjiang** [tog-ETH-er/togETHer#6] [General]
Agenda:

- Community Updates
- Eth Taipei visit
- Fireside discussion

会议决议：

以后的会议是否录制:

**结论同意**

参与投票人员如下：
沙漏
hao yun
Esther
Harry
Frozen
Emily
shooter
Riely
Stepheny

**以下问题再议**
分享平台
分享方式

会议主题：togETHer
会议时间：2020/12/28 21:00-22:00
重复周期：每两周 (周一)

点击链接入会，或添加至会议列表：
https://meeting.tencent.com/s/LxNajKWJw4RV

会议 ID：891 5098 6775

手机一键拨号入会
+8675536550000,,89150986775#(中国大陆)
+85230018898,,,2,89150986775#(中国香港)

根据您的位置拨号
+8675536550000(中国大陆)
+85230018898(中国香港)

**Comment by 0x-stan** [comment]
# Dapp-Learning

近两周工作进展

## repo update

:tada: 2022年1月11日，达到1k star :tada:

![image](https://user-images.githubusercontent.com/19644290/150728440-8fa9e17e-e0ce-4dc2-b557-ae6e5be72878.png)


19个PR #556 ～ #575

https://github.com/Dapp-Learning-DAO/Dapp-Learning/pulls

## sharing meetup

6场分享会，完美收官

- Euler.finance & PID Controlled IR
  - by DL
  - https://youtu.be/WV1B_hUXXFU
  - https://youtu.be/X76qM216aYg
- Ethereum Yellow Paper 内部讨论
  - by BugWriter
  - https://youtu.be/tIwAURd_WUQ
  - https://youtu.be/FG3FWraDJTQ
  - https://youtu.be/yjnviwjKy5g
- Juicebox 项目分享
  - by Roshan
  - https://youtu.be/a0Qr4ffnDu4
- 《主权个人》读书交流会
  - by Alex（中文译者）
  - https://youtu.be/f-Hcd_Mfcxo
  - https://youtu.be/xoXF9WFFxko
- 去中心化期权
  - by Kai
  - https://youtu.be/AS82ADniJqE
- LayerZero项目分析
  - by LonersLiu
  - https://youtu.be/-f3kCqgOUVc
 
提案对新的一批优秀的分享者发放奖励1000元(160DAI)

https://github.com/Dapp-Learning-DAO/DLIP/issues/5

过年期间，分享会将暂停两周

## 翻译小组

- MIT公开课翻译第8集进入排版阶段
- MIT公开课翻译第9/10集进入审稿阶段

**Reply by esther-ecn** [reply]
很密集的分享会啊，赞

**Reply by 0x-stan** [reply]
😄

**Comment by esther-ecn** [comment]
### ECN近期工作内容：
1. 翻译内容输出
2. 春节活动策划
- 社区设计 NFT 新春礼物；POAP
- Vitalik 2月4日社区直播
（如有社区宣传或贡献的渠道及想法，欢迎各位加入联动）

**Comment by Riely007** [comment]
**以太坊翻译小组**

1. [ethereum.org 翻译](https://crowdin.com/project/ethereum-org/zh-CN)
2. [EIP 翻译](https://github.com/ethlibrary/EIPs)

**Comment by luckyyang** [comment]
### Rebase

- 迁移网站 server 到腾讯东京 http://rebase.network/
- [组织分享 algorand 共识算法](https://www.bilibili.com/video/BV1L3411Y7dc/)
- 申请 uniswap hackathon grant
- [Rostra](https://rostra.notion.site) sprint 3 开发

**Comment by winkybupt** [comment]
## 原语里弄
1. Lane's Choice 日历包含了原语里弄搜集的公开的、可参与的线上活动，现在公测期间，感兴趣的可以联系我们获得测试链接。

**Reply by Emilyxiannv** [reply]
最近看到一个区块链日历：Foresight News。类似的？

**Comment by bobjiang** [comment]
HiBlock/Gitcoin/Bob 更新

1. Gitcoin黑客松 - https://gov.gitcoin.co/t/gitcoindao-hackathon-2022/9405
2. EthDenver - Feb 17
3. ENS ecosystem Steward - https://discuss.ens.domains/t/steward-nomination-ens-ecosystem-working-group/9263/5?u=bobjiang
4. Kernel Block 5

**Comment by bobjiang** [comment]
以后的会议是否录制:

结论同意

参与投票人员如下：
沙漏
hao yun
Esther
Harry
Frozen
Emily
shooter
Riely
Stepheny

以下问题再议
分享平台
分享方式

**Comment by Emilyxiannv** [comment]
DAOSquare: 最近做了一次分享，主题是：熊市生存指南，视频有一点没剪完，稍后我会把我们录屏的链接贴在这里。

**Reply by Emilyxiannv** [reply]
熊市生存指南 链接：https://www.youtube.com/watch?v=IO5dEGkgJLw

**Comment by Jackieyewang** [comment]
计划2022年的周期性区块链技术学校的Twitter Space ---- Crypto Speak Low

**Comment by bobjiang** [comment]
sismo dao

https://spruceid.com/
